### ISSUE TITLE
protobuf not getting installed

### ISSUE THREAD

**Issue by mstechly** [quantumlib/Qualtran#517]
Hey!

I have followed the regular installation instruction and when I tried to use serialization module I got the following error:
```
from google.protobuf import descriptor as _descriptor
ModuleNotFoundError: No module named 'google'
```

Installing `protobuf` manually solved the issue, but I just wanted to point out that perhaps something is missing from the installation instructions?

**Comment by mpharrigan** [comment]
fixed by #521 and shouldn't happen again with fixed CI

**Comment by mstechly** [comment]
Awesome, thanks @mpharrigan !
